judgels-frontends/raphael/src/routes/uriel/contests/ContestRole/ContestRoleColor.tsx

```diff
@@ -0,0 +1,10 @@
+import { Intent } from '@blueprintjs/core';
+
+import { ContestRole } from 'modules/api/uriel/contestWeb';
+
+export const contestRoleColor = {
```

Comment:
Let's just refactor the `<Tag .../>` of a contest role into a separate component, instead of just refactoring the color :)